contents/docs/surveys/manual.mdx

```diff
@@ -47,16 +47,72 @@ posthog.getActiveMatchingSurveys((surveys) => {
 
 ```js-web
 posthog.capture("survey sent", {
-    $survey_id: {survey.id} // required
-    $survey_response: {survey_response} // required
-    $survey_name: {survey.name} // optional
+  $survey_id: survey.id, // required
+  $survey_response: 'survey_response', // required
+  $survey_name: survey.name  // optional
 })
 ```
 
 Surveys that are shown should be captured with a "survey shown" event and those that are dismissed, with a "survey dismissed" event similar to the above.
 
 > PostHog currently supports surveys with either a single text input, or a link to another page (such as a form, or scheduling platform), so be sure to use the syntax above in order to track results. We'll be adding more survey types soon!
 
+### Option 3: Backend or API event capture
```

Comment:
This should basically be like option 2 👀 which is API mode, maybe option 2.5? or extension of option 2? 

Basically we just support:

1. do it through posthog's UI and API
or
2. do it through your own UI and API

Reply:
Should we recommend a way to do it from the backend?

Reply:
Also should we recommend always getting the surveys from PostHog first, rather than hard-coding IDs like I did here?

Reply:
> Should we recommend a way to do it from the backend? I guess it is an extension of option 2.

Oh I guess there is a third option of pure backend, where they have to call the `/api/surveys` endpoint and manually match the surveys themselves, because we don't support `getActiveMatchingSurveys` in any libraries other than the JS one for now. This would be the equivalent to how users can call feature flags through the `decide` endpoint

Reply:
I'd be surprised though if someone goes super out of their way to do that ^ for now as the general use case always involves some frontend piece so the JS library support should suffice